### DISCUSSION TITLE
Welcome to MyExpenseBot Discussions!

### DISCUSSION THREAD

**Discussion by Nisarg20** [NCSU-Group70-CSC505-SE-Fall-23/MyExpenseBot#22] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ChaitanyaS182k** [comment]
We are working on adding new features for the upcoming release V5.0.1
- Adding Group Expense Feature
- Adding more currencies for you guys!
- Adding graphs for better visualization.
